src/explore-education-statistics-frontend/src/prototypes/components/PrototypeSearchForm.tsx

```diff
@@ -16,7 +16,7 @@ interface State {
 }
 
 class PrototypeSearchForm extends Component<{}, State> {
-  public state = {
+  public state: State = {
     currentlyHighlighted: undefined,
```

Comment:
Shouldn't have to define state as undefined as we can just avoid including it in the state object.